src/app/settings/organization-plans.component.html

```diff
@@ -219,6 +219,7 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
         <hr class="my-3">
         <h2 class="spaced-header mb-4">{{ (createOrganization ? 'paymentInformation' : 'billingInformation') | i18n}}
         </h2>
+        <p class="text-success" *ngIf="freeTrial && createOrganization">{{ 'freeTrial' | i18n }}</p>
```

Comment:
Was the verbiage for this text directed from the marketing or product team? Just curious because it's repeated again just below the card information in the `'paymentChargedWithTrial'` string key, which is similar but different, just seems a little redundant, or perhaps like we can at least repeat the same message in both places, or move it to the top (and would no longer need the new string/key)...

Reply:
The text I added was stolen directly from the Asana task. One of the images in the task also had different text for the bottom text. I did not copy that one because I don't know if they were aware what exactly it said because of the other bug where it wouldn't show the correct text when routed there. 

I agree that is seems a little redundant. I think that maybe just fixing the routing bug where the correct free trial text will be shown when routed to the page and maybe making the text a tad larger would correct the issue.